examples/advanced/lr-newton-raphson/requirements.txt

```diff
@@ -0,0 +1,3 @@
+flamby @ git+https://github.com/owkin/FLamby.git@main
+nvflare-nightly @ git+https://github.com/NVIDIA/NVFlare.git@main
```

Comment:
I removed line 2 for installing nvflare, since this is mentioned in the [setup instructions](https://github.com/NVIDIA/NVFlare/tree/main/examples#set-up-a-virtual-environment).